src/test/resources/features/cs/ManagementUI.feature

```diff
@@ -233,12 +233,47 @@ Feature: ManagementUI
 
 #    ADD A USER
 
+  @UserManagement
+  Scenario: A new user can be created in DECS through User Management
+    Given I select to "Add a user"
+    When I enter the details for the new user
```

Comment:
Can this be 'I submit the details for the new user'? 'Enter', to me, sounds like you are just typing.